specs/www.rst

```diff
@@ -121,12 +121,8 @@ The *connection scope* information passed in ``scope`` contains:
   listening port, or ``[path, None]`` where ``path`` is that of the
   unix socket. Optional; if missing defaults to ``None``.
 
-Servers are responsible for handling inbound and outbound chunked transfer
-encodings. A request with a ``chunked`` encoded body should be automatically
-de-chunked by the server and presented to the application as plain body bytes;
-a response that is given to the server with no ``Content-Length`` may be chunked
-as the server sees fit.
```

Comment:
Why was this removed? :thinking:

Reply:
Mistake, thank you for pointing it out 😄